aioredis/commands/sentinel.py

```diff
@@ -0,0 +1,140 @@
+import asyncio
+
+from aioredis.util import wait_ok
+
+
+def nativestr(x):
```

Comment:
We can get rid of this by passing `encoding` parameter to `execute` method and fixing `util.decode` function to recursively decode lists